Dockerfile

```diff
@@ -61,6 +61,6 @@ RUN go install -tags=xversion -ldflags="\
       -X tailscale.com/version.GitCommit=$VERSION_GIT_HASH" \
       -v ./cmd/...
 
-FROM alpine:3.11
+FROM alpine:latest
```

Comment:
Using "latest" means this isn't reproducible. We can't bisect to a prior Tailscale commit and get a reproducible Docker build.

Can you just bump this up to a new version instead? (that's effectively how go.mod etc work)